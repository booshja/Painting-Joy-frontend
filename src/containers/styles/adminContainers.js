// dependencies
import styled from "styled-components";
// breakpoints
import { breakpoints } from "../../breakpoints";

export const StyledCell = styled.div`
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    border: 1px solid #207070;
    width: 100%;
    padding: 1rem;
    background-color: #ffffff;

    ${breakpoints("flex-basis", "%", [{ 1024: 50 }])}
`;
