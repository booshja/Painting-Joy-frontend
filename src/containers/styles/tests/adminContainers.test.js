import { render } from "@testing-library/react";
import { StyledCell } from "../adminContainers";

const child = <h1>This is a test</h1>;

/** Smoke Test */
it("renders without crashing", () => {
    render(<StyledCell>{child}</StyledCell>);
});

/** Snapshot Test */
it("matches snapshot", () => {
    const { asFragment } = render(<StyledCell>{child}</StyledCell>);
    expect(asFragment()).toMatchSnapshot();
});

/** RTL Tests */
it("renders correctly", () => {
    const { getByText } = render(<StyledCell>{child}</StyledCell>);

    getByText("This is a test");
});
