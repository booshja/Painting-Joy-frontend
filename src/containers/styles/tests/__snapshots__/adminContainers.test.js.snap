// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdnxRM gklqQU"
  >
    <h1>
      This is a test
    </h1>
  </div>
</DocumentFragment>
`;
